### ISSUE TITLE
pre-trained model is useful.

### ISSUE THREAD

**Issue by zuoxiang95** [keithito/tacotron#85]
Recently, I try to use pre-trained model, and i get some nice result.
Firstly, i trained model with a dataset which contains 20 hours wavs. The model was trained 400k steps. Then i continue to train with another dataset which only has 1.5 hours. And model is only trained 5k steps, it can get some good result. BTW, two datasets are both girl's speech audio.

I hope someone can provide the pre-trained model , that we can get result more quickly.

Here is my result audio.

[eval.zip](https://github.com/keithito/tacotron/files/1562446/eval.zip)

**Comment by HaoyuanLuo** [comment]
I've also trained a Chinese Model with the 25 hours' voice data I got from XimalayaFM. But the quality of my model is much worse than your perfect one.
I found that no matter how long the model is trained, the voice  is always not as smooth as I heared from these samples from the issues both in English and Chinese. Besides some unexpected synthesis results. Would you like to share some advice on my problem?

**Comment by zuoxiang95** [comment]
I think the quality of your dataset is very important. And you can use shengmu and yunmu to be the symbols.It can make model converge more quickly.

**Comment by willswong11** [comment]
Working on the same problem...but lack of enough data. Is the pre-trained model you mentioned also trained on Chinese?

**Comment by zuoxiang95** [comment]
@willswong11 Yes, the quality of data that used to train pre-trained model is not very good.

**Comment by begeekmyfriend** [comment]
Hey @zuoxiang95 , I heard out that the `THCHS30` is the dataset you have adopted for Chinese mandarin training. I have also worked out a pre-trained model and open it at #118 . What I am interested in are as follows:
- 20 hours wav files are not the full data in `THCHS30`. Which files are did you use for training?
- Which type of symbols did you adopt for mandarin text? [pinyin characters](https://github.comhttps://github.com/begeekmyfriend/tacotron/blob/phone_symbols/text/symbols.py#L9/begeekmyfriend/tacotron/blob/master/text/symbols.py#L12) as [a-z12345] or [phoneme sequences](https://github.com/begeekmyfriend/tacotron/blob/phone_symbols/text/symbols.py#L9)?

Thanks in advance.

**Comment by v-yunbin** [comment]
@ begeekmyfriend when download you pre-model, loss the web site and download failture, why?

**Comment by wengsiangchan** [comment]
> I've also trained a Chinese Model with the 25 hours' voice data I got from XimalayaFM. But the quality of my model is much worse than your perfect one.
> I found that no matter how long the model is trained, the voice is always not as smooth as I heared from these samples from the issues both in English and Chinese. Besides some unexpected synthesis results. Would you like to share some advice on my problem?

Hey @zuoxiang95 , I'm new to voice synthesis. I knew that you had successfully trained a Chinese model. May i know the dataset for your model is collected manually or it is already available? Because I wish to train a new model using self-collected dataset. Can you share what is the item we should lookout when training a new model?
